AdyenCardScanner/Sources/CardScanner.swift

```diff
@@ -9,13 +9,31 @@ import UIKit
 
 public typealias CardScanDetails = (number: String?, expirationDate: Date?)
 
+/// A protocol that defines an interface for presenting a user interface to scan payment cards.
```

Comment:
`A protocol that defines an interface for presenting a view controller to scan payment cards.`